schema/deploy/mutations/update_form_change.sql

```diff
@@ -9,8 +9,10 @@ $$
 
   update cif.form_change set
     new_form_data = coalesce(form_change_patch.new_form_data, new_form_data),
-    operation = coalesce(form_change_patch.operation, operation),
-    validation_errors = coalesce(form_change_patch.validation_errors, validation_errors),
+    operation = case
+      when form_change_patch.operation is null and operation = 'archive' then 'update'
+      else coalesce(form_change_patch.operation, operation)
+    end,    validation_errors = coalesce(form_change_patch.validation_errors, validation_errors),
```

Comment:
I don't totally understand what the card is asking for, so I'm not sure if this change covers everything. In this sentence:
> When the update_form_change() mutation is passed an operation as part of the patch, then it will update the operation to 'update'. When not passed 'update' as the operation, the coalesce in the mutation defaults back to 'archive'.

Is that saying that regardless of what operation `update_form_change` receives, it will always default to "update"? In which case I think that needs addressing?

Reply:
My understanding is that quote is describing current behaviour.  Passing in `UPDATE` or `undefined`/`null` provide different results.

The expected behaviour is that regardless of explicitly stating we want an `UPDATE` or passing in nothing we should get the same result, which is an `UPDATE`.

This is due to the `coalesce` where the first parameter will be `NULL` and second will be `ARCHIVE` so we need to conditionally check for this unique case.